services/self_hosted.py

```diff
@@ -44,16 +45,16 @@ def admin_owners() -> QuerySet:
 
 def is_admin_owner(owner: Optional[Owner]) -> bool:
     """
-    Returns true iff the given owner is an admin.
+    Returns true if the given owner is an admin.
     """
     return owner is not None and admin_owners().filter(pk=owner.pk).exists()
 
 
-def activated_owners() -> QuerySet:
-    """
-    Returns all owners that are activated in ANY org's `plan_activated_users`
-    across the entire instance.
-    """
+async def activated_owners_async() -> QuerySet:
+    return await sync_to_async(activated_owner_query)()
+
+
+def activated_owner_query() -> QuerySet:
```

Comment:
should we be worried about perf at all with this fn? genuinely curious

Reply:
yes 🙃 
I gated it behind a license check - so it will only run if the license is valid